src/pages/Details.js

```diff
@@ -1,38 +1,65 @@
 import { useParams } from "react-router-dom";
 import { useNavigate } from "react-router-dom";
 import styled from "styled-components";
+import { useState } from "react";
 
-export default function Details({ output }) {
+export default function Details({
+  deleteFeedback,
+  allDataItems,
+  editFeedback,
+}) {
   const navigate = useNavigate();
 
   const { id } = useParams();
-  const details = output.filter((obj) => obj.id === +id);
 
-  /* console.log(output);
-  console.log(details);
-  console.log(id); */
+  const details = allDataItems.filter((obj) => obj.id === +id);
+
+  const [toggleFeedback, setToggleFeedback] = useState(false);
 
   return (
     <StyledDetails>
       <figure>
         <img src={details[0].image} alt="schuh"></img>
```

Comment:
You might want to check for the existence of details[0]. If a user navigates to a previous non existent id the page will crash.

Reply:
actually i wanted to put an object with the same id in const details. there is only one that can be found and it is on  the position of 0. So i take details[0].  
Can you explain please what do you mean under "If a user navigates to a previous non existent id the page will crash"? What should i click to prove it (crash)?

Reply:
Just navigate to a non existent id (route "/42" for example) manually over the browser and it will crash. You might not be able to achieve this right now by clicking current links on your app, but chances are it will happen in the future with users being able to set bookmarks etc. However I think of it as optional for you right now to fix.

Reply:
Thank you Thomas, i understand now what you mean and try to fix it later!